### PR TITLE
Move stuff to make it available for salt-ssh

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
Move some stuff from call_high to call_chunks so that it is available to salt-ssh.

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/39239

### Tests written?

No

### CODE REVIEW

**Comment by gtmanfred** [comment]
@garethgreenaway can you take a look at this since I moved the disabled stuff to call_chunks?

Thanks!
Daniel

**Comment by gtmanfred** [comment]
Also, looking for opinions on this.  It can easily be moved to Develop if we decided to just put this change into nitrogen.

**Comment by garethgreenaway** [comment]
@gtmanfred quick glance looks good.

**Comment by gtmanfred** [comment]
moving to develop

**Comment by gtmanfred** [comment]
Moving this over here https://github.com/saltstack/salt/pull/39907

File: salt/state.py [hunk 1]
```diff
@@ -2257,35 +2282,11 @@ def call_high(self, high):
...
-                        self.__run_num += 1
-                        chunks.remove(low)
-                        break
-
         # If there are extensions in the highstate, process them and update
         # the low data chunks
         if errors:
             return errors
-        ret = dict(list(disabled.items()) + list(self.call_chunks(chunks).items()))
+	ret = self.call_chunks(chunks)
```
Comment (s0undt3ch) [inline salt/state.py]: Tabs? Missing a space?

**Pull request closed without merging**